client/environment.go

```diff
@@ -94,6 +94,7 @@ type Environment struct {
 	TerragruntWorkingDirectory  string        `json:"terragruntWorkingDirectory,omitempty"`
 	VcsCommandsAlias            string        `json:"vcsCommandsAlias"`
 	BlueprintId                 string        `json:"blueprintId" tfschema:"-"`
+	IsRemoteBackend             *bool         `json:"isRemoteBackend" tfschema:"-"`
```

Comment:
`tfschema:"-"` means that we want to remove it from the schema. 
It's used for schema auto-assignment that's done in `writeResourceData`

i'm not 100% sure what adding the `tfschema:"-"` would affect, but I think that it would mean that drifts won't be detected